packages/web/config.ts

```diff
@@ -17,7 +17,13 @@ export const CONFIG = {
     process.env.NEXT_BRIGHTID_NODE_URL || 'http:%2f%2fnode.brightid.org',
   publicURL: process.env.NEXT_PUBLIC_FRONTEND_URL,
   gaId: process.env.NEXT_PUBLIC_GA4_ID,
-  clarityId: process.env.NEXT_CLARITY_ID,
+  honeybadgerApiKey: process.env.NEXT_PUBLIC_HONEYBADGER_API_KEY,
+  honeybadgerReportData:
+    process.env.NEXT_PUBLIC_HONEYBADGER_REPORT_DATA === 'true',
+  honeybadgerDebug: process.env.NEXT_PUBLIC_HONEYBADGER_DEBUG === 'true',
+  userbackToken:
+    process.env.NEXT_PUBLIC_USERBACK_TOKEN ||
+    '37882|74240|mjn1zXGiYaozZR6qmqFfjl9Sk',
```

Comment:
These will need to be added to the dockerfile and github action as well I believe

Reply:
Correct. I did post in builders guild but a better place to mention it would have been in this PR. 😅

Reply:
I think adding these to the `docker/frontend/Dockerfile` should be part of this PR. You could then build the container locally and see if the variables make it in properly. LMK if you need help with that, some trial and error is probably involved. If it works in the PR deployment we should be good

Reply:
no, these would go in the frontend container which is not included in the docker-compose. So you'd have to manually `docker build` it. That only gets you part of the way anyway, probably the best thing to do would be to just push what you have. The test is if those env variables end up in the PR deployment on Google cloud

Reply:
> probably the best thing to do would be to just push what you have. 

Pushing now.